modules/olm-installing-opm.adoc

```diff
@@ -30,26 +37,46 @@ $ podman login registry.redhat.io
 ----
 endif::[]
 
-. Extract the `opm` binary from the Operator Registry image and copy it to your
-local file system:
+. Extract the `opm` binary for the operating system you want from the Operator
+Registry image and copy it to your local file system.
 +
-[source,terminal]
-ifdef::openshift-origin[]
+--
+* For Linux:
++
+[source,terminal,subs="attributes+"]
 ----
-$ oc image extract quay.io/openshift/origin-operator-registry:4.6.0 \
+$ oc image extract {registry-image} \
+ifndef::openshift-origin[]
+    -a ${REG_CREDS} \
+endif::[]
     --path /usr/bin/opm:. \
     --confirm
 ----
-endif::[]
-ifdef::openshift-enterprise,openshift-webscale[]
+
+* For macOS:
++
+[source,terminal,subs="attributes+"]
 ----
-$ oc image extract registry.redhat.io/openshift4/ose-operator-registry:v4.6 \
-    -a ${REG_CREDS} \//<1>
-    --path /usr/bin/opm:. \
+$ oc image extract {registry-image} \
+ifndef::openshift-origin[]
+    -a ${REG_CREDS} \
+endif::[]
+    --path /usr/bin/darwin-amd64-opm:. \
     --confirm
 ----
-<1> Specify the location of your registry credentials file.
+
+* For Windows:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc image extract {registry-image} \
+ifndef::openshift-origin[]
+    -a ${REG_CREDS} \
 endif::[]
+    --path /usr/bin/windows-amd64-opm:. \
+    --confirm
+----
+--
 
 . Make the binary executable:
```

Comment:
@adellape  for the following step, better to indicate that it take linux or mac as example because "chmod +x ./opm" and "sudo mv ./opm /usr/local/bin/" are not OK on windows.